test/smitten/smitten.ts

```diff
@@ -48,7 +50,7 @@ describe('hoe', () => {
     const f = e.of('F')
     e.emit.call(null, 100)
     f.emit.call(null, 200)
-    assert.deepEqual(actions, [100, action('F', 200)])
+    assert.deepStrictEqual(actions, [100, action('F', 200)])
```

Comment:
how is this part of lint changes?

Reply:
lint threw an error for using deprecated method `deepEqual` and suggested the change to `deepStrictEqual`